package.json

```diff
@@ -70,6 +80,10 @@
     "*.js": [
       "prettier --config .prettierrc.json --write",
       "git add"
+    ],
+    "*.md": [
+      "prettier --no-config --tab-width 4 --print-width 120 --write --prose-wrap always",
```

Comment:
This command on *.md is the one achieving "The code snippets within the markdown are formatted (on Husky commit)"? What about the other mentioned in the PR body "The markdown is nicely formatted. (on Husky commit)" ?

Reply:
This command on `*.md` is the one achieves both.

* It nicely formats the **markdown** as **markdown***


Something like:

```
indented markdown      with      excess spaces 
* item one
- item two
  * item 2a
  * item 2b
```

Will be changed into:

```
indented markdown with  excess spaces 
-   item one
-   item two
    -  item 2a
    -  item 2b
```
Properly indented, consistent markup


* It formats the **codesnippets within the markdown** according to the **text highlighting  indicator**

```
Some markdown text:
```javascript
   crampedFunction(){var x; return dothis();}
``
```

```
Some markdown text:

```javascript
crampedFunction (){
    var x; 
    return dothis();
}
``
```

In this case the snippet is recognised as **JavaScript** and formatted accordingly

Reply:
NTC